content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -246,6 +246,15 @@ Source errors include code-level information about the error. More information a
 | `action.target.name` | string | Element that the user interacted with. Only for automatically collected actions. |
 | `action.name` | string | User-friendly name created (for example, `Click on #checkout`). For [Custom User Actions][15], the action name given in the API call. |
 
+### Frustration signals fields
+
+| Field                | Type   | Description                                                   |
+|-------------------------------|--------|---------------------------------------------------------------|
+| `session.frustration.count`     | number | Count of all frustration signals associated with one session. |
+| `view.frustration.count`        | number | Count of all frustration signals associated with one view.    |
+| `action.frustration.type:dead`  | string | The dead clicks detected by the RUM Browser SDK.              |
```

Comment:
for all of the action.frustration.type they need to have a click after. so for example, it's action.frustration.type:dead_click. this naming convention should follow on the other. two action types as well